src/graphics.rs

```diff
@@ -284,15 +284,15 @@ impl Window {
     /// - `ylabel` is y axis title
     /// - `zlabel` is z axis title
     pub fn set_axes_titles(&mut self, xlabel: String, ylabel: String, zlabel: String) {
-        let cprops = &Cell {row: self.row, col: self.col, title: String::from(""), cmap: self.cmap};
-        let xstr = CString::new(xlabel.as_bytes()).unwrap();
-        let ystr = CString::new(ylabel.as_bytes()).unwrap();
-        let zstr = CString::new(zlabel.as_bytes()).unwrap();
+        let cprops = &Cell {row: self.row, col: self.col, title: [0].as_ptr(), cmap: self.cmap};
```

Comment:
Can you please use [std::ptr::null](https://doc.rust-lang.org/std/ptr/fn.null.html) instead of an static single element slice to get a invalid pointer. Please do similar change in all other places where `[0].as_ptr()` is being used.